Trackovid19-web/src/index.html

```diff
@@ -9,7 +9,7 @@
     <link rel="apple-touch-icon" sizes="180x180" href="assets/img/favicon/apple-touch-icon.png" />
     <link rel="icon" type="image/png" sizes="32x32" href="assets/img/favicon/favicon-32x32.png" />
     <link rel="icon" type="image/png" sizes="16x16" href="assets/img/favicon/favicon-16x16.png" />
-    <link rel="manifest" href="assets/img/favicon/site.webmanifest" />
+    <link rel="manifest" href="manifest.webmanifest" />
     <link rel="mask-icon" href="assets/img/favicon/safari-pinned-tab.svg" color="#5bbad5" />
     <meta name="msapplication-TileColor" content="#da532c" />
```

Comment:
This meta tag content should be updated to the project orange color `#fa6400`

Reply:
Same as previous, but I will update to white

Reply:
The `msapplication-TileColor` in particular shouldn't be white since the Microsoft tiles text are white as well.

Reply:
I will revert it to the color that designer give me from the beginning